Fix grammar in HTML label element

CliqLabs opened the pull request:
Made some minor grammar fixes to improve clarity.

<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description
Made a couple of minor grammar fixes to improve clarity.

<!-- ✍️ Summarize your changes in one or two sentences -->

### Motivation
Confusing sentence structures make concepts more difficult to understand.  I've reworded a few sentences so they're easier to parse.

<!-- ❓ Why are you making these changes and how do they help readers? -->

### Additional details


<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

CliqLabs committed:
Update index.md

Made some minor grammar fixes to improve clarity.

Josh-Cena commented on files/en-us/web/html/element/label/index.md:
```diff
@@ -80,7 +80,7 @@ There are no special styling considerations for `<label>` elements — structura
 ### Interactive content
 
-Don't place interactive elements such as {{HTMLElement("a", "anchors")}} or {{HTMLElement("button", "buttons")}} inside a `label`. Doing so makes it difficult for people to activate the form input associated with the `label`.
+Do not place interactive elements such as {{HTMLElement("a", "anchors")}} or {{HTMLElement("button", "buttons")}} inside a `label`. Doing so makes it difficult to activate the form input associated with the `label`.
```
No need to expand contractions: they are generally encouraged in technical writing to make it sound casual.

Josh-Cena commented on files/en-us/web/html/element/label/index.md:
```diff
@@ -13,10 +13,10 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
 Associating a `<label>` with a form control, such as {{htmlelement("input")}} or {{htmlelement("textarea")}} offers some major advantages:
 
-- The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
+- The label text is not only visually associated with its corresponding text input; it is programmatically associated with it, too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
```
```suggestion
- The label text is not only visually associated with its corresponding text input; they are programmatically associated, too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
```

Josh-Cena reviewed (commented):
More drive-by

CBID2 commented on files/en-us/web/html/element/label/index.md:
```diff
@@ -13,10 +13,10 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
 - When a user clicks or touches/taps a label, the browser passes the focus to its associated input (the resulting event is also raised for the input). That increased hit area for focusing the input provides an advantage to anyone trying to activate it — including those using a touch-screen device.
 
-To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, you add the `for` attribute to the `<label>` element, where the value of `for` is the same as the `id` in the `<input>` element.
+To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, add the `for` attribute to the `<label>` element, where the value of `for` is the same as the `id` in the `<input>` element.
```
```suggestion
To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, add the `for` attribute to the `<label>` element. Lastly, make the value of `for` the same as the `id` in the `<input>` element.
```

CBID2 reviewed (commented):
Hi @CliqLabs! I just made a suggestion to the tutorial on working with`<label>` and `for` elements. I hope it helps! 😄

CliqLabs committed:
Update files/en-us/web/html/element/label/index.md

Thank you very much for your help 🙏

(For what it's worth, I don't even know how this all happened.  I made a few suggestions a month or two ago; fast forward to today and the next thing I know it seems I have at least committer privileges(?).  I honestly don't know how this aspect of Github really works.  I'm just shrugging my shoulders and saying 'okay' at this point, lol.)

Co-authored-by: Christine Belzie <105683440+CBID2@users.noreply.github.com>

queengooborg committed:
Merge branch 'main' into patch-1

hamishwillee commented:
Hi @CliqLabs - there are some drive-by comments from Josh above - can you please check them?

bsmth commented:
Thanks for the submission @CliqLabs - I'm going to close this one due to inactivity and unaddressed review comments. Let us know if you'd like to come back to this and we can reopen it.

Thank you :)
